### PR TITLE
CheckboxWidget and other gui related mappings

### PR DESCRIPTION
**Pull request by NeunEinser**
A check box widget was added for the "erase cache data" option inside the optimize world screen.

### CODE REVIEW

**Comment by Runemoro** [comment]
Shouldn't it be "Checkbox" rather than "CheckBox"?
 - https://en.wikipedia.org/wiki/Checkbox (uses checkbox throughout the article)
 - `textures/gui/checkbox.png` (not `check_box`)
 - https://www.google.com/search?q=%22checkbox%22 (34 million results) vs https://www.google.com/search?q=%22check+box%22 (16 million results)

**Comment by NeunEinser** [comment]
Hmm, I am used to the capital B, programming languages I use always do that, e.g.

https://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/CheckBox.html
https://docs.oracle.com/javase/7/docs/api/javax/swing/JCheckBox.html
https://docs.microsoft.com/de-de/dotnet/api/system.windows.forms.checkbox

But I don't really care tbh.

File: mappings/net/minecraft/client/gui/widget/LevelListWidget.mapping [hunk 1]
```diff
@@ -11,6 +11,11 @@ CLASS dcr net/minecraft/client/gui/widget/LevelListWidget
 			ARG 2 levelList
 			ARG 3 level
 			ARG 4 levelStorage
+		METHOD a play ()V
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/widget/LevelListWidget.mapping:14]: Hmm - what do these methods do?

**Review by asiekierka — APPROVED** [review]

File: mappings/net/minecraft/client/gui/widget/LevelListWidget.mapping (hunk 1 above)
Comment (Runemoro) [reply mappings/net/minecraft/client/gui/widget/LevelListWidget.mapping:14]: I think they're called when the user clicks play, for example.

Comment (NeunEinser) [reply mappings/net/minecraft/client/gui/widget/LevelListWidget.mapping:14]: Correct. It's what gets executed, when you press `play` on the level select screen.

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**